docs/source/includes/validate.rst

```diff
@@ -1,6 +1,7 @@
-The purpose of the validate command is to determine whether topologies are syntactically valid and, if not, provide a list of errors that occured during validation
+Validate Command
+-----------------
 
-Using the validate command
+The purpose of the validate command is to determine whether topologies and layouts are syntactically valid.  If not, it will provide a list of errors that occured during validation
 ```````````````````````````
```

Comment:
If you leave the backticks in here, it will attempt to make the lines above it a title.